WalletWasabi.Packager/Program.cs

```diff
@@ -200,6 +200,9 @@ private static async Task SignAsync()
 		// Verify back the signature file.
 		await WasabiSignerHelpers.VerifySha256SumsFileAsync(sha256sumAscFilePath);
 
+		// Verify back Wasabi installer's hashes
+		await VerifyInstallerFileHashesAsync(finalFiles, sha256SumsFilePath);
```

Comment:
Add `.ConfigureAwait(false);` and to the line above as well.

Reply:
In theory it should work without a problem, just a re-check, but it's yet to be tested on the release machine.

Reply:
Can you test it? You can generate fake installation files and run the packager with argument -sign